### ISSUE TITLE
KeyError: 'id'

### ISSUE THREAD

**Issue by rajadain** [WikiWatershed/model-my-watershed#2413]
View details in Rollbar: [https://rollbar.com/Azavea/MMW/items/235/](https://rollbar.com/Azavea/MMW/items/235/)

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/rest_framework/views.py", line 452, in dispatch
    response = self.handle_exception(exc)
  File "/usr/local/lib/python2.7/dist-packages/rest_framework/views.py", line 449, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/rest_framework/decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "/opt/app/apps/bigcz/views.py", line 170, in values
    return Response(_get_values(request))
  File "/opt/app/apps/bigcz/views.py", line 150, in _get_values
    return values(**values_kwargs)
  File "/opt/app/apps/bigcz/clients/cuahsi/details.py", line 55, in values
    return wof.get_values(wsdl, site, variable, from_date, to_date)
  File "/usr/local/lib/python2.7/dist-packages/ulmo/cuahsi/wof/core.py", line 185, in get_values
    values = waterml.v1_1.parse_site_values(response_buffer)
  File "/usr/local/lib/python2.7/dist-packages/ulmo/waterml/v1_1.py", line 16, in parse_site_values
    methods=methods)
  File "/usr/local/lib/python2.7/dist-packages/ulmo/waterml/common.py", line 52, in parse_site_values
    values_element, metadata_elements, namespace)
  File "/usr/local/lib/python2.7/dist-packages/ulmo/waterml/common.py", line 241, in _parse_metadata
    for item in collection
KeyError: 'id'
```

**Comment by rajadain** [comment]
Don't think we can do much about this error. Probably just need to wrap this in a `try` block and return a managed exception code.

**Comment by rajadain** [comment]
This should be fixed in #2466 since we now use an ulmo fork that catches this: https://github.com/emiliom/ulmo/commit/90dbfe31f38a72ea4cee9a52e572cfa8f8484adc
